### PR TITLE
HPCC-16370 Adds whitelist entry to dropzone entries

### PR DESCRIPTION
**Pull request by rpastrana**
- Provides mechanism to specify un-configured DZs as "trusted" DZs

Signed-off-by: Rodrigo Pastrana <rodrigo.pastrana@lexisnexis.com>

### CODE REVIEW

**Comment by rpastrana** [comment]
@wangkx please review.

File: initfiles/componentfiles/configxml/dropzone.xsd.in [hunk 1]
```diff
@@ -88,6 +88,37 @@
...
+						<xs:appinfo>
+							<viewChildNodes>true</viewChildNodes>
+						</xs:appinfo>
+					</xs:annotation>
+					<xs:complexType>
+						<xs:sequence>
+							<xs:element name="WhiteListEntry" type="xs:string" minOccurs="0" maxOccurs="1"/>
+						</xs:sequence>
+					</xs:complexType>
+					<xs:attribute name="server" type="string" use="optional">
```
Comment (wangkx) [inline initfiles/componentfiles/configxml/dropzone.xsd.in]: Is it possible to define a DZ without a network address?

File: initfiles/componentfiles/configxml/dropzone.xsd.in [hunk 2]
```diff
@@ -88,6 +88,37 @@
...
+						</xs:attribute>
+						<xs:attribute name="name" type="xs:string" use="required">
+                            <xs:annotation>
+                                <xs:appinfo>
+                                    <colIndex>1</colIndex>
+                                </xs:appinfo>
+                            </xs:annotation>
+                        </xs:attribute>
+				</xs:element>
+			</xs:sequence>
```
Comment (wangkx) [inline initfiles/componentfiles/configxml/dropzone.xsd.in]: Incorrect indent

**Review by wangkx — COMMENTED** [review]
@rpastrana 2 comments

**Comment by rpastrana** [comment]
@wangkx both issues addressed. Please review.

**Comment by wangkx** [comment]
@rpastrana looks fine.

**Comment by rpastrana** [comment]
@richardkchapman ready to merge

**Pull request merged**